README.md

```diff
@@ -1,19 +1,13 @@
-##homebrew-formulae
-
-This [Homebrew](http://brew.sh) tap provides homebrew formulae for packages from the mbed team.
-
-### Using This Tap
-First, [install homebrew](http://brew.sh):
+## Homebrew formulae for Arm Mbed
 
+[Homebrew](https://brew.sh) is a package manager for macOS. You can use it to install [the Arm GCC toolchain](https://developer.arm.com/open-source/gnu-toolchain/gnu-rm):
 ```sh
-ruby -e "$(curl -fsSL https://raw.github.com/Homebrew/homebrew/go/install)"
+brew tap ArmMbed/homebrew-formulae
+brew install arm-none-eabi-gcc
 ```
 
-Then tap this repository:
-```sh
-brew tap ARMmbed/homebrew-formulae
-```
+We update this tap with new releases of the toolchain whenever they become available.
 
-NOTE: if you are using Github 2-factor Auth, you have to use an OAuth token as your password if prompted at the command line during login to Github by Homebrew. Create a token at https://github.com/settings/applications under Personal Access Tokens.
+This tap is maintained by the Arm Mbed team, not the Arm OSS compiler team! Please raise [issues with this formula here](https://github.com/ArmMbed/homebrew-formulae/issues), and [issues with the toolchain here](https://bugs.launchpad.net/gcc-arm-embedded).
 
-Then you can `brew install <things>` from this tap.
+NOTE: if you are using Github 2-factor Auth, you have to use an OAuth token as your password if prompted at the command line during login to Github by Homebrew. Create a token at https://github.com/settings/applications under Personal Access Tokens.
```

Comment:
When would anybody be prompted for password? This. is. public. repo.

Reply:
Good point, fixed!